ui/src/views/Rules.vue

```diff
@@ -2140,6 +2416,15 @@ export default {
 </script>
 
 <style>
+.blue-background {
+  background-color: lightblue;
```

Comment:
We can use the specific light color provided by Patternfly: https://www.patternfly.org/v3/styles/color-palette/ the code color is `#00b9e4`